### PR TITLE
Make all tc execute via tash command

### PR DESCRIPTION
**Pull request by junmin-kim**
If CONFIG_TASH is enabled, tc_main installs tc commands.
Else tc_main make task and run tc.

### CODE REVIEW

**Comment by seinfra** [comment]
Code Rule Check OK.

**Comment by seinfra** [comment]
Code Rule Check OK.

File: apps/examples/testcase/tc_main.c [hunk 1]
```diff
@@ -84,18 +85,79 @@ extern int utc_audio_main(int argc, char *argv[]);
 extern int itc_audio_main(int argc, char *argv[]);
 
 /* Not yet */
-extern int mpu_tc_main(int argc, char *argv[]);
+extern int tc_mpu_main(int argc, char *argv[]);
+
+#ifdef CONFIG_TASH
+const static tash_cmdlist_t tc_cmds[] = {
+#ifdef CONFIG_EXAMPLES_TESTCASE_ARASTORAGE_UTC
+	{"arastorage_utc", utc_arastorage_main, TASH_EXECMD_SYNC},
```
Comment (sunghan-chang) [inline apps/examples/testcase/tc_main.c]: ```SYNC``` -> ```ASYNC``` for all of tc commands.

File: apps/examples/testcase/tc_main.c [hunk 2]
```diff
@@ -25,6 +25,7 @@
 #include <sched.h>
 #include <sys/types.h>
 #include <semaphore.h>
+#include <apps/shell/tash.h>
 
 #if defined(CONFIG_EXAMPLES_TESTCASE_ARASTORAGE_UTC) || defined(CONFIG_EXAMPLES_TESTCASE_ARASTORAGE_ITC)
 #define TC_ARASTORAGE_STACK       4096
```
Comment (sunghan-chang) [inline apps/examples/testcase/tc_main.c]: When CONFIG_TASH is enabled, these definitions are not needed. Let's check a conditional.

**Comment by seinfra** [comment]
Code Rule Check OK.

**Comment by seinfra** [comment]
Code Rule Check OK.

**Pull request merged**